cmd/oras/internal/display/status/deprecated.go

```
@@ -58,7 +58,8 @@ var printer = NewPrinter(os.Stdout)
 
 // Print objects to display concurrent-safely.
 func Print(a ...any) error {
-	return printer.Println(a...)
+	printer.Println(a...)
+	return nil
```

Need documentation here to explain why and how the print errors are ignored.